executor.ts

```diff
@@ -52,11 +59,11 @@ export class Executor {
   async init(): Promise<void> {
     tracer.open('Executor#init');
     // Ensure the commander's state is clean before initializing.
-    await this.close();
+    await this.close(false);
```

Comment:
Could also make this happen inside `using` but not in `close`. But I think I'd rather wait to refactor this until I consider adding a separate session abstraction.